### ISSUE TITLE
Feature request: configurable tracking of branches

### ISSUE THREAD

**Issue by llakala** [mynixos/support#17]
Currently, MyNixOS only tracks 24.05. While this is understandable, it causes a lot of confusion for users who are looking for a package that's been added to unstable, yet they can't find it when searching. 

Ideally, a user would be able to choose the branch they're tracking. I think a GUI option for this makes the most sense: the amount of things that can be done text-only in MyNixOS is wonderful, but a text option is inherently hidden from the user unless used. Some form of dropdown for selecitng a branch would be perfect. Tracking the latest stable and unstable branch is enough, in my opinion.

If this won't be added, a relevant error message when no packages are found could be nice, saying "Your package may not be on an unstable branch, which MyNixOS is currently unable to track".

**Comment by pveierland** [comment]
Agreed! MyNixOS 2.0 will provide improved support for branch + commit tracking.
